stdlib/public/core/UnsafeRawBufferPointer.swift.gyb

```diff
@@ -644,21 +652,26 @@ extension Unsafe${Mutable}RawBufferPointer {
       return (it, UnsafeMutableBufferPointer(start: nil, count: 0))
     }  
 
+    _debugPrecondition(_end != nil)
+    var newCount = 0
     for p in stride(from: base, 
       // only advance to as far as the last element that will fit
-      to: base + count - elementStride + 1, 
+      // We unsafely unwrap because _end cannot be nil if _base is not.
```

Comment:
Nit: The internal invariant check above already tells me it's okay to use the bogus unwrap. If the note on `_base != nil` ⟹ `_end != nil` needs to be noted, then it ought to be noted on that invariant check.

```suggestion
```